src/components/Icon.tsx

```diff
@@ -0,0 +1,7 @@
+interface IIconProps {
+    img: string;
+    className: string;
```

Comment:
do we need className?

Reply:
Each Icon used differently so it has it own classname.